src/analysis/retail/warlock/demonology/modules/features/SummonDemonicTyrant.tsx

```diff
@@ -1,143 +1,150 @@
+          this._summsWithDemonicPower[index]['Total'] * 10 + '%';
+      });
+      RoTAvgBuff = this._summsWithDemonicPower[0]['RoT Buff'];
+    }
+
+    let row = 0;
+    const petTableRows = Object.keys(this._summsWithDemonicPower[0]).map((demonSource) => {
+      row += 1;
       return (
-        <tr key={demonSource}>
-          <td align="left">
-            <SpellLink spell={Number(demonSource)} />
-          </td>
-          <td align="center">
-            {(mergedPets[Number(demonSource)] / this._petsPerCast.length).toFixed(2)}
-          </td>
+        <tr key={'row' + demonSource} className={row % 2 ? 'tyr-odd-row' : ''}>
+          <td className="tyr-align-left">{demonSource}</td>
+          {Object.entries(this._summsWithDemonicPower).map(([key, tyrCastSumms]) => (
+            <td key={demonSource + key}>{tyrCastSumms[demonSource]}</td>
+          ))}
         </tr>
       );
     });
 
-    const avgTyrantPower =
-      this.demonicTyrantPower.reduce((acc, val) => acc + val, 0) /
-      Math.max(this.demonicTyrantPower.length, 1);
-    const tyrantFooter = this._hasReignOfTyranny
-      ? `Average Reign of Tyranny Bonus Damage: ${avgTyrantPower.toFixed(2)}%`
-      : null;
-
-    const petTable =
-      this._petsPerCast.length > 0 ? (
-        <>
-          <thead>
-            <tr>
-              <th>Pet Source</th>
-              <th>Avg Pets per Cast</th>
-            </tr>
-          </thead>
-          <tbody>{petTableRows}</tbody>
-        </>
-      ) : null;
+    const petTable = (
+      <table className="tyr-table">
+        <thead>
+          <tr>
+            <th style={{ textAlign: 'left' }}>Tyrant Cast</th>
+            {Object.keys(this._summsWithDemonicPower).map((key) => (
+              <th className="tyr-align-center" key={'tyrcast' + key}>
+                {Number(key) !== 0 ? Number(key) : 'Average'}
+              </th>
+            ))}
+          </tr>
+        </thead>
+        <tbody>{petTableRows}</tbody>
+      </table>
+    );
+
+    const avgDemonsEmpowered = Number(this._summsWithDemonicPower[0]['Total']);
+    const avgWildImpsEmpowered = Number(this._summsWithDemonicPower[0]['Wild Imp']);
+
     return (
-      <StatisticBox
+      <Statistic
         position={STATISTIC_ORDER.CORE(6)}
         size="flexible"
         tooltip="Number of pets empowered by each Demonic Tyrant summon."
-        value={avgPets.toFixed(2)}
         dropdown={petTable}
-        footer={tyrantFooter}
       >
         <BoringSpellValueText spell={SPELLS.SUMMON_DEMONIC_TYRANT}>
-          {`${avgPets.toFixed(2)}`} <small>Avg. demons empowered</small>
+          {avgDemonsEmpowered.toFixed(1)} <small>Avg. demons buffed</small>
+          <br />
+          {this._hasReignOfTyranny && <small>{RoTAvgBuff} Avg. RoT bonus dmg</small>}
+          {this._hasReignOfTyranny && <br />}
+          <small>
+            {avgWildImpsEmpowered.toFixed(1)}/{this._hasReignOfTyranny ? 15 : 10} Avg. imps buffed
+          </small>
```

Comment:
prefer block or paragraph elements to <br/> for spacing. e.g.

```suggestion
          <p>{avgDemonsEmpowered.toFixed(1)} <small>Avg. demons buffed</small></p>
          {this._hasReignOfTyranny && <p><small>{RoTAvgBuff} Avg. RoT bonus dmg</small></p>}
          <small>
            {avgWildImpsEmpowered.toFixed(1)}/{this._hasReignOfTyranny ? 15 : 10} Avg. imps buffed
          </small>
```